application/views/_template.php

```diff
@@ -3,31 +3,32 @@
 	<head>
 		<title>{title}</title>
 		<meta HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=UTF-8"/>
-		<link href="../../css/Global.css" rel="stylesheet" type="text/css"/>
-		<link href="../../css/Header_Footer.css" rel="stylesheet" type="text/css"/>
+		<link href="../../css/style.css" rel="stylesheet" type="text/css"/>
+		<link href="../../css/button.css" rel="stylesheet" type="text/css"/>
+		<link href="../../css/tablet.css" rel="stylesheet" type="text/css"/>
+		<link href="../../css/smartphone.css" rel="stylesheet" type="text/css"/>
 	</head>

```

Comment:
For all your scripts, could you try using `(strlen(dirname($_SERVER['SCRIPT_NAME'])) === 1 ? "/" : dirname($_SERVER['SCRIPT_NAME']) . "/") . "css"` instead of using `../../` since this doesn't work when trying to access via `localhost/gamebot5`.